docs/extensions/extensions_manifest.md

```diff
@@ -1,37 +1,40 @@
 ---
-title: Manifest Format
+title: Extensions Manifest Format
 category: 62c5cb137dbdad00536291a6
 hidden: true
 ---
 
+## Overview
+
 Extension metadata is defined in the `PhylumExt.toml` file, using the [TOML]
 format. Every manifest file consists of the following sections:
 
- - [`name`] — Extension name
- - [`entry_point`] — Execution entry point
- - [`permissions`] — Permissions required for execution
-     - [`[[read]]`] — Required read path permissions
-     - [`[[write]]`] — Required write path permissions
-     - [`[[env]]`] — Required environment variable permissions
-     - [`[[run]]`] — Required process execution permissions
-     - [`[[net]]`] — Required network domain permissions
+- [`name`] — Extension name
+- [`entry_point`] — Execution entry point
+- [`permissions`] — Permissions required for execution
+  - [`[[read]]`][read] — Required read path permissions
+  - [`[[write]]`][write] — Required write path permissions
+  - [`[[env]]`][env] — Required environment variable permissions
+  - [`[[run]]`][run] — Required process execution permissions
+  - [`[[net]]`][net] — Required network domain permissions
```

Comment:
This still may not be the best way to represent the TOML tables, but the old way was definitely wrong (see the current formatting here - <https://docs.phylum.io/docs/manifest_format>). The changes here create the desired links and looks like this:

---

<img width="950" alt="image" src="https://user-images.githubusercontent.com/18729796/179868027-4869f052-7374-41af-af6a-3e80815b8ebb.png">

Reply:
I've followed [cargo manifest format](https://doc.rust-lang.org/cargo/reference/manifest.html) for documenting things here. I find their methods to be best at getting the point across in list form.

Reply:
ah...that cargo format is nice.

If the understanding is correct, then the only "table" that can exist in an extension manifest is `[permissions]`. The permission types (`read`, `write`, `env`, `run`, and `net`) are all just key-value pairs, correct? If the `[permissions]` table is allowed to show up multiple times in a manifest, then it can be represented as an array of tables (e.g, `[[permissions]]`).

Reply:
The most recent changes cause the output to look like this:

---

<img width="950" alt="image" src="https://user-images.githubusercontent.com/18729796/180045380-29a7fe41-6adb-4294-82cb-ad13613abf96.png">

Reply:
Why did you remove the array double brackets?

Reply:
`permissions` appears to be a [table](https://toml.io/en/v1.0.0#table), specified only one time in the manifest. If `permissions` can be...or is expected to be...specified more than one time in the manifest, then it would be an [array of tables](https://toml.io/en/v1.0.0#array-of-tables).

Reply:
Good point, the distinction between "array" and "array of tables" makes sense.